schemas/json/block.json

```
@@ -377,6 +377,11 @@
 					"type": "boolean",
 					"default": true
 				},
+				"visibility": {
+					"description": "By default, a block can be visibility by a user from the block 'Options' dropdown. To disable this behavior, set visibility to false.",
```

```suggestion
					"description": "By default, a block can be hidden by a user from the block 'Options' dropdown. To disable this behavior, set visibility to false.",
```

"Can be visibility" is grammatically incorrect, so let's correct it.